README.rdoc

```diff
@@ -1,28 +1,44 @@
 == README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Are you at Campina Grande/PB during São João and are looking for the events of Parque do Povo?
```

Comment:
Do you want to stay tuned about the big events that will be happening at Parque do Povo during the Greatest São João of the World - Campina Grande?